### PR TITLE
Remove unnecessary check for certChain parameter since it can be empty

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

**Comment by lookuptable** [comment]
You'll need to make the file reading optional here: https://github.com/istio/auth/blob/master/cmd/istio_ca/main.go#L162. Otherwise I believe it'll error out.

**Comment by wattli** [comment]
Good catch, updated.

File: cmd/istio_ca/main.go [hunk 1]
```diff
@@ -158,13 +158,18 @@ func createCA() ca.CertificateAuthority {
 		return ca
 	}
 
+	certChainBytes := []byte{0}
```
Comment (lookuptable) [inline cmd/istio_ca/main.go]: nit: just do `var certChainBytes []byte`

**Comment by lookuptable** [comment]
\lgtm

**Pull request merged**